packages/ddp-server/livedata_server.js

```diff
@@ -1521,79 +1536,30 @@ _.extend(Server.prototype, {
     });
   },
 
-  call: function (name /*, arguments */) {
-    // if it's a function, the last argument is the result callback,
+  call: function (name, ...args) {
+    // If it's a function, the last argument is the result callback,
     // not a parameter to the remote method.
-    var args = Array.prototype.slice.call(arguments, 1);
-    if (args.length && typeof args[args.length - 1] === "function")
-      var callback = args.pop();
-    return this.apply(name, args, callback);
+    return (args.length &&
+            typeof args[args.length - 1] === "function")
+      ? this.apply(name, args, args.pop())
```

Comment:
I'm not really a fan of this code style - I think breaking it up into multiple logical lines makes more sense to me. What really tripped me up is that we are calling pop at the same time as passing the array into apply.

But that's just my opinion, so feel free to ignore.